ret-core/src/main/kotlin/io/rabobank/ret/RetContext.kt

```diff
@@ -2,6 +2,16 @@ package io.rabobank.ret
 
 import io.quarkus.runtime.annotations.RegisterForReflection
 
+/**
+ * Context of the current RET execution (to be injected)
+ *
+ * This class is used to pass information about the context to plugins. Those in turn can use it to base logic upon.
+ * Note that this class needs to be registered for reflection in your plugin.
```

Comment:
Without reading more context, is this clear what this means? How would one do this in their plugin?

Reply:
I've now also described which annotation should be added to the entry command